ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -1364,6 +1364,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-marketplace
+- as: azure-aks-hypershift-guest-f7
+  cron: 24 8 3,10,17,26 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      HYPERSHIFT_HC_VERSION: "4.16"
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~NonPreRelease&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&;~Serial&;~Disruptive&
+      TEST_TIMEOUT: "30"
+    test:
+    - chain: openshift-e2e-test-hypershift-qe
+    workflow: cucushift-installer-rehearse-azure-aks-hypershift-guest
```

Comment:
It looks like we only have the CI for guest. Don't we need to add the CI for mgmt as well?

Reply:
I discussed it with @fxierh , the mgmt is not an OCP cluster, so we would not add mgmt ci this time.
Just avoid spending too much effort to adjust the OCP cases in the aks.